framework/src/outputs/TableOutput.C

```diff
@@ -34,6 +34,9 @@ InputParameters validParams<TableOutput>()
   InputParameters params = validParams<AdvancedOutput<FileOutput> >();
   params += AdvancedOutput<FileOutput>::enableOutputTypes("postprocessor scalar vector_postprocessor");
 
+  // Option for writing vector_postprocessor time file
+  params.addParam<bool>("time_data", false, "When true and VecptorPostprocessor data exists, write a csv file containing the timestep and time information.");
```

Comment:
Let me comment to make sure I understand the design:
Normally each vector postprocessor rights it's own CSV file, one row per timestep. Currently there is no time, or timestep information in that file.

Question: I see two designs, either add this information to every vector postprocessor or create a separate file with that information. You've gone with the second and it does have the advantage that it does not duplicate information. It has the drawback that any tool that analyzes this information needs to open two files.

Is that all correct?